db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180321125113) do
+ActiveRecord::Schema.define(version: 2018_05_01_155601) do
```

Comment:
How come the date format has changed?

Reply:
I wondered about this too, I'm not sure either!